test/Ocelot.AcceptanceTests/ServiceHandler.cs

```diff
@@ -46,6 +46,27 @@ public void GivenThereIsAServiceRunningOn(string baseUrl, string basePath, Reque
             _builder.Start();
         }
 
+        public void GivenThereIsAServiceRunningOn(string baseUrl, string basePath, long maxBodySize, RequestDelegate del)
+        {
+            _builder = new WebHostBuilder()
+                .UseUrls(baseUrl)
+                .UseKestrel()
+                .ConfigureKestrel(options =>
```

Comment:
Seems we have another service handlers which have Kestrel options.
So we need to merge such helpers, otherwise this copied-pasted code will never end.
So, better to have delegate argument than passing a set og them!

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1972/commits/dcbbab7119180345afaf001fbf3664f64b2fafd7 ✔️